### DISCUSSION TITLE
Implement new Strict Tool Call support

### DISCUSSION THREAD

**Discussion by Bodekaer** [RageAgainstThePixel/com.openai.unity#274] [Q&A] [answered]
@StephenHodgson have you seen the new `Strict` option for function calls, and any plans to implement this?
Will be helpful, as I randomly experience hallucinated function calls :) 

More info:
https://platform.openai.com/docs/guides/function-calling/function-calling-with-structured-outputs

**Comment by StephenHodgson** [comment, marked as answer]
Yes I already have a task to complete this

- https://github.com/RageAgainstThePixel/OpenAI-DotNet/issues/343
